tools/pepsi/src/pre_game.rs

```diff
@@ -96,6 +96,10 @@ pub async fn pre_game(arguments: Arguments, repository: &Repository) -> Result<(
     .await
     .wrap_err("failed to set player numbers")?;
 
+    wireless(WirelessArguments::Scan { naos: naos.clone() })
+        .await
+        .wrap_err("failed to scan for networks")?;
+
```

Comment:
Shouldn't this be part of the `prepare` block?
Otherwise you can't `--prepare` if not all robots are reachable.

Reply:
Good point, fixed now.